Serializer/Normalizer/FlattenExceptionHandler.php

```diff
@@ -94,7 +96,7 @@ private function convertToArray(FlattenException $exception, Context $context):
     {
         if ($context->hasAttribute('status_code')) {
             $statusCode = $context->getAttribute('status_code');
-        } else {
+        } elseif (false === $statusCode = $this->statusCodeMap->resolveFromClassName($exception->getClass())) {
```

Comment:
@olix21 When testing this patch in an application using FOSRestBundle 3.0, you need to replace `false` with `null` here:

```php
} elseif (null === $statusCode = $this->statusCodeMap->resolveFromClassName($exception->getClass())) {
```

Reply:
I didn't have to change `false` to `null` to make it work